apps/examples/vite.config.ts

```diff
@@ -0,0 +1,23 @@
+import path from 'path'
+import { defineConfig } from 'vite'
+
+export default defineConfig({
+	root: path.join(__dirname, 'src'),
+	publicDir: path.join(__dirname, 'public'),
+	resolve: {
+		alias: [
+			{
+				find: '@tldraw/',
+				replacement: path.resolve(__dirname, '../packages/'),
+			},
+		],
+	},
```

Comment:
I don't think this config is required. The way the packages are set up it should and use the local source .ts(x) files by default.

I just tried it without and it seemed fine, but maybe i'm missing some nuance.